CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fix CoreDNSMaxHPAReplicasReached alert to not fire in case max and min are equal. 
 
+### Removed
+
+- Remove Alertmanager ingress
```

Comment:
```suggestion
- Remove AlertManager ingress to avoid conflicts with the existing one, until the new AlertManager is ready to replace the one from _g8s-prometheus_
```